### PR TITLE
Update maintained/v5.0.x (#443) 

### PR DESCRIPTION
**Pull request by fdobrovolny**
Issue #443

### CODE REVIEW

**Comment by fdobrovolny** [comment]
The test results without cilium enabled is following:

```
Sonobuoy has completed. Use `sonobuoy retrieve` to get results.                                                                                                                                                                               
=== Collecting results ===                                                                                                                                                                                                                    
INFO[0000] delete request issued                         dry-run=false kind=namespace namespace=sonobuoy                                                                                                                                      
INFO[0000] delete request issued                         dry-run=false kind=clusterrolebindings names="[sonobuoy-serviceaccount-sonobuoy]"                                                                                                    
INFO[0000] delete request issued                         dry-run=false kind=clusterroles names="[sonobuoy-serviceaccount-sonobuoy]"                                                                                                           
INFO[0000] delete request issued                         dry-run=false kind=namespace namespace=svcaccounts-7354                                                                                                                              
Plugin: e2e                                                                                                                                                                                                                                   
Status: passed                                                                                                                                                                                                                                
Total: 7069                                                                                                                                                                                                                                   
Passed: 361                                                                                                                                                                                                                                   
Failed: 0                                                                                                                                                                                                                                     
Skipped: 6708                                                                                                                                                                                                                                 
                                                                                                                       
Plugin: systemd-logs                                                                                                   
Status: passed                                       
Total: 3                                                                                                               
Passed: 3                                                                                                              
Failed: 0                                                                                                              
Skipped: 0                                                                                                             
                                                                                                                       
Run Details:
API Server version: v1.25.11         
Node health: 3/3 (100%)
Pods health: 29/34 (85%)                         
Details for failed pods:
svcaccounts-7354/pod-service-account-defaultsa Ready:False: ContainersNotReady: containers with unready status: [token-test]
svcaccounts-7354/pod-service-account-defaultsa-mountspec Ready:False: ContainersNotReady: containers with unready status: [token-test]
svcaccounts-7354/pod-service-account-mountsa Ready:False: ContainersNotReady: containers with unready status: [token-test]
svcaccounts-7354/pod-service-account-mountsa-mountspec Ready:False: ContainersNotReady: containers with unready status: [token-test]
svcaccounts-7354/pod-service-account-nomountsa-mountspec Ready:False: ContainersNotReady: containers with unready status: [token-test]
```

File: Release-Notes-R5.md [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
 
+## Updated software
+
+### sonobuoy 0.56.x
+
+- Sonobuoy v0.56.17 adds support for the latest k8s versions even for k8s 1.27.
+
+### cert-manager 1.12.x
+### kind 0.18.0
+### helm 3.12.x
```
Comment (chess-knight) [inline Release-Notes-R5.md:11]:
```suggestion
```

**Review by chess-knight — COMMENTED** [review]
Overall LGTM, only minor changes, please

File: Release-Notes-R5.md (hunk 1 above)
Comment (chess-knight) [reply Release-Notes-R5.md:11]: Those are changes from #437 #438 #439 which are not included here.

**Review by chess-knight — APPROVED** [review]

**Review by matofeder — APPROVED** [review]
lgtm

**Comment by jschoone** [comment]
Hi @matofeder and @chess-knight, could one of you merge this?

**Pull request merged**